src/nvim/ui_bridge.c

```diff
@@ -181,12 +182,19 @@ static void ui_bridge_cursor_goto_event(void **argv)
 
 static void ui_bridge_cursor_styleset(UI *b, Dictionary args)
 {
-  UI_CALL(b, cursor_styleset, 2, b, &args);
+  Object copy = copy_object(DICTIONARY_OBJ(args));
+  Object *a = xmalloc(sizeof(Object));
+  *a = copy;
+  UI_CALL(b, cursor_styleset, 2, b, a);
```

Comment:
who else is using `args`? Normally you should just pass a pointer and it is only freed when `ui_bridge_cursor_styleset_event` is reached.